src/util/config.cpp

```diff
@@ -90,17 +90,17 @@ bool configt::set(const cmdlinet &cmdline)
   if(cmdline.isset("function"))
     main = cmdline.getval("function");
 
```

Comment:
Don't change these flags, they should be equal to how compilers handle them.

Reply:
From users side this will not make difference they can type -I, -D, -f or -W.
The change is inside the code as boost doesn't accept it. It should be written as string and letter like "include,I".
This was the solution to not change your interface.

Reply:
I see, as long as it doesn't change our cmdline interface, then it's fine.